WalletWasabi.Fluent/Helpers/AssetHelpers.cs

```diff
@@ -6,16 +6,21 @@ namespace WalletWasabi.Fluent.Helpers;
 
 public static class AssetHelpers
 {
-	public static Bitmap GetBitmapAsset(Uri uri)
+	public static Bitmap? GetBitmapAsset(Uri uri)
 	{
 		var assets = AvaloniaLocator.Current.GetService<IAssetLoader>();
-		using var image = assets.Open(uri);
-		return new Bitmap(image);
+
+		if (assets is { })
+		{
+			using var image = assets.Open(uri);
+			return new Bitmap(image);
+		}
+
+		return null;
 	}
```

Comment:
Instead of allowing null, it should throw an exception... Allowing null here causes allowing null at many other places.

Reply:
https://docs.microsoft.com/en-us/uwp/api/Windows.UI.Xaml.Data.IValueConverter?view=winrt-22000

UnsetValue should be returned from an IValueConverter implementation that provides conversion in a data binding to a dependency property, in any case where the converter is unable to convert a source value. Converters shouldn't throw exceptions for that case in Convert; these will surface as run-time exceptions that you'd need to add handling for in UnhandledException or worse yet appear to users as actual run-time exceptions. Converter implementations should follow the general binding pattern that any failed binding does nothing and does not provide a value, and UnsetValue rather than null is the sentinel value for that case that the binding engine understands. For more info, see Data binding in depth.

iv made our converters in this PR compliant with those docs.